views/partials/footer.ejs

```diff
@@ -10,4 +12,5 @@
     <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
 </body>
-</html>
\ No newline at end of file
+</footer>
```

Comment:
first `footer` tag closes and then `body`, not vice versa.

Reply:
@thebishaldeb see my latest commit. I guess commit no. 4

Reply:
commit : 6b3545a
https://github.com/NJACKWinterOfCode/Instances/pull/42/commits/6b3545a17d7fcf143b18384a920f0066726071de